pyvrp/search/__init__.py

```diff
@@ -17,7 +17,7 @@
 from .neighbourhood import NeighbourhoodParams as NeighbourhoodParams
 from .neighbourhood import compute_neighbours as compute_neighbours
 
-NODE_OPERATORS = [
+NODE_OPERATORS: list[NodeOperator] = [
```

Comment:
Mypy complains about this but I don't understand why.

Reply:
These are classes, not instantiated objects. This should work: `list[Type[NodeOperator]]` (using `typing.Type`).